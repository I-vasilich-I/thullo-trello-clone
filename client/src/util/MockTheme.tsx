import { ReactNode } from 'react';
import { theme } from 'src/styles/theme.style';
import { ThemeProvider } from 'styled-components';

interface Props {
  children: ReactNode;
}

const MockTheme: React.FC<Props> = ({ children }) => {
  return <ThemeProvider theme={theme}>{children}</ThemeProvider>;
};

export default MockTheme;
