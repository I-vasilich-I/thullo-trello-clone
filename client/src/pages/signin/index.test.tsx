import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import MockTheme from 'src/util/MockTheme';
import SingIn from '.';

describe('Sign in page', () => {
  it('render sign in page unchanged', () => {
    const { container } = render(
      <MockTheme>
        <SingIn />
      </MockTheme>,
    );

    expect(container).toMatchSnapshot();
  });
});
