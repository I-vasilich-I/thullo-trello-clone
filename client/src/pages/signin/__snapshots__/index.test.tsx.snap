// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign in page render sign in page unchanged 1`] = `
<div>
  <section>
    Sing in
  </section>
</div>
`;
